emu/src/memory/internal_memory.rs

```diff
@@ -110,7 +110,7 @@ impl IoDevice for InternalMemory {
             0x04000200..=0x040003FE => self.interrupts.write_at(address, value),
             0x05000000..=0x050001FF => self.bg_palette_ram[address - 0x05000000] = value,
             0x05000200..=0x050003FF => self.obj_palette_ram[address - 0x05000200] = value,
-            0x06000000..=0x06017FFF => self.video_ram[address - 0x06000000] = value,
+            0x06000000..=0x06FFFFFF => self.video_ram[address - 0x06000000] = value,
```

Comment:
VRAM is ` 06000000-06017FFF   VRAM - Video RAM          (96 KBytes)` no?

Reply:
Yes, I extended to also NotUsed part because I want to avoid other problem of read/write on non used memory

Reply:
I think it would be better to put not_used explicitly instead of increasing the vram